phobos/cap-vars.yml

```diff
@@ -0,0 +1,54 @@
+---
+log_flavor: vm-osa-aio
+log_image: a4a9bc69-de27-4cec-903a-cf57be693738
+log_count: 1
+
+stor_flavor: ironic-rpc-storage-cap
+stor_image: 78856fa6-1de4-42d3-b545-466f6501e21f
+
+security_group: 58753c65-1804-4eee-8349-77f1c5573001
+external_network: ironic
+networks:
+  - net-name: "{{ external_network }}"
+
+fiobench_pgnum: 2048
+fiobench_size: 100G
+sata_bench: false
+ssd_bench: false
+bench_rgw_user_password: rgwbenchsecret
+bench_rgw_hummingbird_url:  https://github.com/troubling/hummingbird/releases/download/v1.0.0/hummingbird
+
+monitor_interface: bond0
+public_network: 172.20.40.0/22
+cluster_network: 172.20.40.0/22
+osd_scenario: non-collocated
+devices:
+  - /dev/sdd
+  - /dev/sde
+  - /dev/sdf
+  - /dev/sdg
+  - /dev/sdh
+  - /dev/sdi
+  - /dev/sdj
+  - /dev/sdk
+
+dedicated_devices:
+  - /dev/sdb
+  - /dev/sdb
+  - /dev/sdb
+  - /dev/sdb
+  - /dev/sdc
+  - /dev/sdc
+  - /dev/sdc
+  - /dev/sdc
+
+external_lb_vip_address: "{{ bootstrap_host_public_address | default(ansible_default_ipv4.address) }}"
+internal_lb_vip_address: 172.20.41.111
+haproxy_default_services:
```

Comment:
Do we need to specify this? If so the defaults aren't working properly and we should find out why, I imagine it could be because we are not linking the group_vars into the playbook dir.
If so, lets fix that instead of duplicating the haproxy_default_services var.